api/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.56.2"
```

Comment:
I'm thinking we should move toward using https://pypi.org/project/setuptools-scm/ to get the version number from the git tag instead of maintaining the version in two places: in this setup file and in git tags. Right now it's possible to update one and forget to update the other like we saw with v1.56.2:

* https://github.com/gro-intelligence/api-client/pull/149 (version number updated on development branch on 2019-12-23)
* https://github.com/gro-intelligence/api-client/releases/tag/v1.56.2 (version not tagged/released until 2020-01-07)

Would be better to have a single source of truth. Can experiment with that on Monday.

Reply:
done
I hardcoded the name 'gro' in client/lib.py when getting the version but I am not sure if there is any other way to fetch the name defined in setup.py